devtools/story-loader.js

```diff
@@ -0,0 +1,64 @@
+/**
+copied and updated from https://github.com/izhan/storybook-description-loader
+will replace once repo accepts pull request + publishes
+ */
+const babel = require('@babel/core');
+
+function createDescriptionNode(name, description) {
+    return babel.types.expressionStatement(
+        babel.types.assignmentExpression(
+            '=',
+            babel.types.memberExpression(babel.types.identifier(name), babel.types.identifier('parameters')),
+            babel.types.objectExpression([
+                babel.types.objectProperty(
+                    babel.types.identifier('docs'),
+                    babel.types.objectExpression([
```

Comment:
I'm debating if I want to edit this further to merge anything already existing in `docs: {` - which is why funamental-styles can't use the jsdoc notes

Reply:
Here's the problem I'm talking about: https://astexplorer.net/#/gist/82ff0a74e4b30a07715683ba12ffe65b/d0afb2869797329284c7b30faf1f10f75735e620